.github/workflows/app-test-build-deploy.yaml

```
@@ -131,6 +132,7 @@ jobs:
       # build the app and deploy it
       - if: github.event_name != 'pull_request'
         name: 'build app for ${{ matrix.os }}'
+        timeout-minutes: 30
```

30 seconds may be too low based on preliminary results from #9949

```suggestion
        timeout-minutes: 60
```